src/components/tx/security/safenet/index.tsx

```diff
@@ -80,6 +69,36 @@ function _getSafeTxHash({ safe, chainId, safeTx }: Required<SafenetTxSimulationP
   })
 }
 
+const StatusAction = ({ status, link }: { status: string; link?: string }): ReactElement => {
+  if (status === 'success') {
+    return (
+      <div>
+        <SvgIcon
+          component={CheckIcon}
+          inheritViewBox
+          fontSize="small"
+          color="success"
+          className={css.safenetCheckIcon}
+        />
+        <span className={css.labelSuccess}>No issues found</span>
+      </div>
+    )
+  } else if (status === 'pending' && link) {
+    return (
+      <Button variant="outlined" size="small" href={link} sx={{ width: '100%', py: 0.5 }}>
+        Share verification link
+      </Button>
```

Comment:
> so the user stays on the transaction page if he clicks.

target blank will open a new tab (so the user will not stay on the tx page)

I think the way to do is here is to use a copy to clipboard button + display the link as text because sometimes the clipboard can not function properly (privacy settings etc)

Reply:
I decided to go with `target='_blank'` for simplicity's sake. I also changed the wording a bit, since opening a window isn't "sharing a verification link":

I also added an icon to indicate it is opening a new page:

![image](https://github.com/user-attachments/assets/fc13eba7-a0fa-4078-946f-f5d2f0fe3b2f)